src/mitol/payment_gateway/api.py

```diff
@@ -757,3 +760,187 @@ def _get_decision_from_response(self, response):
             return ProcessorResponse.STATE_PENDING
 
         return ProcessorResponse.STATE_ERROR
+
+    def find_transactions(self, transactions: List[str], limit=20):
+        """
+        Performs a search for the transactions specified. For simplicity, this
+        assumes the data set specified is order IDs. If your system doesn't
+        produce unique order IDs (or if they get reused for whatever reason),
+        this will likely return multiple transactions for the same order ID.
+
+        Args:
+        - transactions (list of strings): List of order IDs to look for
```

Comment:
Might be helpful to change this to 'reference numbers' or have whatever terminology used here match what is also used in find_and_get_transactions since they both use that same parameters